vendor/bat-native-ads/include/bat/ads/public/interfaces/ads.mojom

```diff
@@ -10,6 +10,16 @@ enum BraveAdsEnvironment {
   DEVELOPMENT
 };
 
+struct BraveAdsInternalsInfo {
```

Comment:
As above, as discussed we should be serializing as JSON in the library and rendering the JSON in the UI as this will reduce changes within the rewards code for future additions